cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,84 @@
 /// <reference types='cypress' />
 
 describe('Web Tables page', () => {
-  it('', () => {
+  let user;
 
+  beforeEach(() => {
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+    });
+    cy.visit('https://demoqa.com/webtables');
+  });
+  it('should check the pagination', () => {
+    cy.addNewWorker(user);
+    cy.addNewWorker(user);
+    cy.addNewWorker(user);
+    cy.get('select').select('5');
+    cy.get('div.-next').click();
+    cy.get('input[aria-label="jump to page"][type="number"][value="2"]')
+      .should('have.value', 2);
+  });
+
+  it('should cout rows', () => {
+    cy.get('select').select('20');
+    cy.get('.rt-tr-group').should('have.length', 20);
+  });
+
+  it('Should add new worker', () => {
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').type(user.userName);
+    cy.get('#lastName').type(user.userSurname);
+    cy.get('#userEmail').type(user.email);
+    cy.get('#age').type('50');
+    cy.get('#salary').type(user.number);
+    cy.get('#department').type(user.department);
+    cy.get('#submit').click();
+    cy.contains(user.email).should('exist');
+  });
+
+  it('should delete worker', () => {
+    cy.addNewWorker(user);
+    cy.contains(user.email).parent('div.rt-tr')
+      .find('.action-buttons [title="Delete"]').click();
```

Comment:
Test case for deleting a worker should include assertions to verify that the worker is no longer present in the table

Reply:
Fixed